File: Java/Kode Program/PetShop.java
class PetShop 
{
    private String Id;
    private String NamaProduk;
    private String KategoriProduk;
    private int HargaProduk;

    // Constructor
    public PetShop(String id, String namaProduk, String kategoriProduk, int hargaProduk) 
    {
        this.Id = id;
        this.NamaProduk = namaProduk;
        this.KategoriProduk = kategoriProduk;
        this.HargaProduk = hargaProduk;
    }

    public PetShop(){

    }

    //Setter dan Getter ID
    void setId(String id)
    {
        this.Id = id;
    }

    String getId()
    {
        return Id;
    }

    //Setter dan Getter Nama Produk
    void setNamaProduk(String nama)
    {
        this.NamaProduk = nama;
    }

    String getNamaProduk()
    {
        return NamaProduk;
    }

    //Setter dan Getter Kategori Produk
    void setKategoriProduk(String kategori)
    {
        this.KategoriProduk = kategori;
    }

    String getKategoriProduk()
    {
        return KategoriProduk;
    }

    //Setter dan Getter Harga Produk
    void setHargaProduk(int harga1)
    {
        this.HargaProduk = harga1;
    }

    int getHargaProduk()
    {
        return HargaProduk;
    }

}

File: Python/Kode Program/PetShop.py
class PetShop:
    def __init__(self, id=None, nama_produk=None, kategori_produk=None, harga_produk=None):
        self.id = id
        self.nama_produk = nama_produk
        self.kategori_produk = kategori_produk
        self.harga_produk = harga_produk

    # Getter dan Setter ID
    def set_id(self, id):
        self.id = id

    def get_id(self):
        return self.id

    # Getter dan Setter Nama Produk
    def set_nama_produk(self, nama):
        self.nama_produk = nama

    def get_nama_produk(self):
        return self.nama_produk

    # Getter dan Setter Kategori Produk
    def set_kategori_produk(self, kategori):
        self.kategori_produk = kategori

    def get_kategori_produk(self):
        return self.kategori_produk

    # Getter dan Setter Harga Produk
    def set_harga_produk(self, harga):
        self.harga_produk = harga

    def get_harga_produk(self):
        return self.harga_produk